examples/src/main/scala/com/example/http4s/ExampleService.scala

```diff
@@ -66,10 +65,7 @@ class ExampleService[F[_]](implicit F: Effect[F]) extends Http4sDsl[F] {
 
       case GET -> Root / "content-change" =>
         // EntityEncoder typically deals with appropriate headers, but they can be overridden
-        val resp: F[Response[F]] = Ok("<h2>This will have an html content type!</h2>")
-        val fr: EffectResponseOps[F] = http4sEffectResponseSyntax(resp)
-        val w: F[Response[F]] = fr.withContentType(Some(`Content-Type`(`text/html`)))
-        w
+        Ok("<h2>This will have an html content type!</h2>", `Content-Type`(`text/html`))
```

Comment:
Whether or not we proceed with this, we should get that other mess out of our example.

Reply:
That looks like some debugging I did the other day...